specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/workspace.json

```diff
@@ -1493,7 +1493,8 @@
           "type": "boolean",
           "x-ms-mutability": [
             "create",
-            "read"
+            "read",
+            "update"
           ]
```

Comment:
Just remove x-ms-mutability.

Reply:
Removing this raises alerts in S360 for ROUNDTRIP_INCONSISTENT_PROPERTY